lib/tran_sock.c

```diff
@@ -391,61 +391,61 @@ tran_sock_init(vfu_ctx_t *vfu_ctx)
 {
     struct sockaddr_un addr = { .sun_family = AF_UNIX };
     tran_sock_t *ts = NULL;
-    int ret;
+    int err;
```

Comment:
Maybe it's best to keep `ret` since `err` is a function defined in `<err.h>`?

Reply:
sure